ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml

```diff
@@ -12,47 +12,52 @@ import StatusQ.Popups 0.1
 StatusModal {
```

Comment:
Could you please use `StatusDialog` as the root component? `StatusModal` is deprecated.

Reply:
@osmaczko done, also linked the corresponding StatusQ PR